App/templates/index.html

```diff
@@ -290,9 +286,9 @@ <h1 class="headroom text-center">metro extracts</h1>
       });
     var cities = countries.selectAll(".city").data(function(d){ return d.metros; });
     cities.enter().append("a").attr("class","city");
-    cities.text(function(d){ return d.name.split("_")[0].replace(/-/g," "); })
+    cities.text(function(d){ return d.name; })
       .attr("href",function(d){ 
```

Comment:
I'm using d3 to keep track of the list divs, so they aren't being shown/hidden they are being data-binded and need to be updated with the current text and href